start-deployCanyon

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+. ${SCRIPTS:-/}start-utils
+isDebugging && set -x
+
+JAVA_VER=$(java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
+
+if [ "${JAVA_VER}" != "8" ]; then
+  log "ERROR: Canyon server type only supports Java version 8"
+  exit 1
+fi
+
+if [ "${VERSION}" != "b1.7.3" ]; then
```

Comment:
I know it's referred to as "Minecraft Beta 1.7.3", but this "b" prefix version convention looks very unusual to me. Can it simply be "1.7.3". Is there a "non beta" 1.7.3? Looks like 1.7.4 was that.

Reply:
According to https://launchermeta.mojang.com/mc/game/version_manifest.json there are two similar version tags: '1.7.3' and 'b1.7.3'. v1.7.3 is a release version of Minecraft as well, so to avoid confusion (and also support the same version tag used by the 'BUKKIT' type and others which work with 'b1.7.3' for Beta 1.7.3) I kept this check to keep consistency.

Reply:
Wow, that was confusing of them. I'm good with "b1.7.3" then.

Reply:
Agreed - there's always been a frustration of mine with the version tags used, but at the very least there is a separation. The above beta tags format work in various `TYPE=BUKKIT` and `TYPE=VANILLA` versions with your docker image already (when the jar url is overridden with many of the repositories providing old alpha/beta server builds of vanilla or craftbukkit).

I might file a Readme PR clarifying setup for those who (inexplicably) want to run very old server version under this image. For now, I think Canyon covers the main use-case and has stability patches and fixes ensuring a smooth run under Java 1.8.